Freshpaint/Classes/FPFreshpaintIntegration.m

```diff
@@ -391,29 +391,46 @@ - (void)notifyForName:(NSString *)name userInfo:(id)userInfo
 
 - (void)sendData:(NSArray *)batch
 {
+    NSString *sessionParameter = [self.analytics validatedSessionId];
+
+    NSMutableArray *mutBatch = [NSMutableArray arrayWithCapacity:batch.count];
```

Comment:
was there not a place before `sendData` where we set properties on the event payload? i would expect a method like `sendData` would only send the event payload as-is; it seems a little out of place to me to also have `sendData` modify the event payload as well.